### PR TITLE
Main

### PR DESCRIPTION
**Pull request by msumro**
### Description

Added support for munch

### Checklist

* [x] Tested on munch
* [x] Usb broke

### CODE REVIEW

**Comment by halal-beef** [comment]
.... what is this

**Review by J0SH1X — CHANGES_REQUESTED** [review]
Please merge this in properly without removing support for other devices.

**Comment by NetuserPRO** [comment]
@msumro fix it

**Comment by msumro** [comment]
@NetuserPRO Edited it

File: Platform/Xiaomi/sm8250/munch.dsc
Comment (qaz6750) [inline Platform/Xiaomi/sm8250/munch.dsc:1]: May I ask why the support for alioth devices has been removed?

File: =
Comment (qaz6750) [inline =:1]: May I ask why there are these junk files ？

File: Platform/Xiaomi/sm8250/Library/munch/PlatformMemoryMapLib/PlatformMemoryMapLib.c
Comment (qaz6750) [inline Platform/Xiaomi/sm8250/Library/munch/PlatformMemoryMapLib/PlatformMemoryMapLib.c:1]: This has nothing to do with adding new device support for you

File: Silicon/Qualcomm/QcomPkg/Include/Library/FrameBufferSerialPortLib.h
Comment (qaz6750) [inline Silicon/Qualcomm/QcomPkg/Include/Library/FrameBufferSerialPortLib.h:1]: This has nothing to do with adding new device support for you

**Review by qaz6750 — CHANGES_REQUESTED** [review]
Please note:
* Your submission contains some junk files
*  Secondly, I don't know why you deleted alioth to support your munch device, which is incorrect
* In addition, this submission contains many changes for other purposes, and you should not include them in this submission, which is very confusing

**Comment by msumro** [comment]
Good, i also deleted that

**Comment by halal-beef** [comment]
@msumro ??

**Pull request closed without merging**